content/docs/04-clusters/02-data-center/01-maas/15-create-manage-maas-clusters.md

```diff
@@ -0,0 +1,155 @@
+| Availability Zones | These are multiple, isolated locations within a Region. When you choose one or more availability zones, Palette provides fault tolerance to guard against hardware failures and network failures by provisioning nodes across availability zones if multiple zones are selected. |
+| Disk Size | Specify the required storage size. |
+
+<br />
+
+
+#### Worker Node Pool
+
+| Property | Description |
+|-----------|-------------|
+| Name | A descriptive name for the node pool. |
+| Size | Number of VMs to be provisioned for the node pool.|
+| Rolling Update | Rolling update has two available options. Review the Update Parameter list below for more details. ?? |
+| [Labels](/clusters/cluster-management/taints#overviewonlabels) | Labels apply placement constraints on a pod. For example, you can add a label to make a node eligible to receive the workload. |
+| [Taints](/clusters/cluster-management/taints#overviewontaints) | Sets toleration to pods and allows (but does not require) the pods to schedule onto nodes with matching taints. |
+| Instance type | The compute instance type to be used for all nodes in the node pool. |
+| Availability Zones | These are multiple, isolated locations within a Region. When you choose one or more availability zones, Palette provides fault tolerance to guard against hardware failures and network failures by provisioning nodes across availability zones if multiple zones are selected. |
+| Disk Size | Specify the required storage size. |
+
+
+12. You can configure the following cluster management features now if needed, or you can do it later:
+
+- Manage Machines
+- Scan Policies
+- Backup Policies
+- Review settings and deploy the cluster. 
+
+13. Review settings and deploy the cluster. 
+
+
+# Validation
+
+You can validate your cluster is available by reviewing the cluster details page. Navigate to the left **Main Menu** and click **Clusters**. The **Clusters** page lists all available clusters Palette manages. Select the cluster to review its details page. Ensure the **Cluster Status** field contains the value **Running**.
+
+
+
+# Delete a MAAS Cluster
+
+When you delete a MAAS cluster, all virtual machines and associated storage disks that were created for the cluster are removed. 
+
+Follow these steps to delete a MAAS cluster.
+
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Navigate to the **Main Menu** and click **Clusters**. 
+
+
+3. Select the cluster you want to delete.
+
+
+4. Click the **Settings drop-down Menu**, and choose **Delete**.
+
+The cluster status is updated to Deleting while cluster resources are being deleted. When all resources are successfully deleted, the cluster status is updated to Deleted and the cluster is removed from the list. The delete operation returns the edge hosts to the **Ready** state. All the artifacts related to the Kubernetes distribution are removed.
```

Comment:
is "Deleted" meant to be capitalised? since it is a status, shouldn't it be listed as code or highlighted in some other way? there is another example later in this paragraph

Reply:
Made **Deleted** and **Deleting** statuses boldface.